File: 8/5/3.cpp
#include <string>
#include <vector>
#include <math.h>
#include <stack>
#include <iostream>

using namespace std;

vector<int> solution(vector<int> progresses, vector<int> speeds) {
    vector<int> answer;
 
    int idx = 0;
    int days = 0;
    while(idx != progresses.size()){
       int count = 0;
       days++;
       for (int i = idx; i < progresses.size(); i++){
              if ((progresses[idx] + (speeds[idx] * days)) >= 100){
                 idx++;
                 count++;
              }
        }
        if (count != 0) 
            answer.push_back(count);
    }
    return answer;
}

File: 2021/2/3/3.py
def solution(n):
    answer = 0

    s = ""
    while n >= 3 :
        s+= str(int(n%3))
        n = n / 3

    s+=str(int(n))

    for i in range(len(s)) :
        answer += int(s[i])*pow(3,len(s)-i-1)


    return answer
